theme/js/components/discussions/thread-reply.vue

```diff
@@ -9,7 +9,15 @@
       </div>
     </div>
     <form class="form fr-p-0" @submit.prevent="submit">
-      <p class="fr-mt-0 fr-mb-1w fr-text--sm">{{$t('* : Required field')}}</p>
+      <i18n-t
+        keypath="Fields preceded by a star ({markup}) are required."
+        tag="p"
+        class="fr-mt-0 fr-mb-1w fr-text--sm"
```

Comment:
Can I suggest having a smaller grey text ? Ex: `fr-text--xs text-grey-380`?

Reply:
I'll try this !

Reply:
I only put `fr-text--xs` for now because I found it hard to read with both. Would it work for you ?